fix: React Native on web should report hardware type as Desktop or Mobile, not Web. (#2769)
PostHog/posthog-js: .changeset/clear-flowers-itch.md

---
'posthog-react-native': patch
---

> fix: on react native running on the web should be Desktop or Mobile and not Web since its a hardware type and not platform

Author: ```suggestion
fix: React Native on web should report hardware type as Desktop or Mobile, not Web
```
